common/pulp/common/constants.py

```diff
@@ -59,13 +59,13 @@
 # Constant used to determine whether a CeleryBeatLock should be removed due to age
 CELERYBEAT_LOCK_MAX_AGE = 200
 # The amount of time in seconds before a Celery process is considered missing
-CELERY_TIMEOUT_SECONDS = 300
+CELERY_TIMEOUT_SECONDS = 60
```

Comment:
I wrote commit, but I meant commit+redmine_ticket. We can implement this entire feature without changing this value. Also the scaling guide I believe talks about failover times which we would need to update. I'm hoping to establish for this PR that we only increase the heartbeat frequency and in a separate PR with a separate issue update the failover speed. What do you all think about that?